capstone-frontend/src/App.jsx

```diff
@@ -42,10 +42,10 @@ class App extends Component {
             <Route exact path='/' component={Home} />
             <Route path='/browse/:query' component={Browse} />
             <Route path='/mybooks' component={MyBooks} />
-            <Route path='/mylists' component={MyLists} />
```

Comment:
We should discuss abandoning this page. I know we added a user's BookLists in the dropdown you created in the sidebar, but it may still be nice to have a page to display all a user's BookLists (similar to the MyClubs page).

Reply:
+1 to this, it could also be used when trying to view another user's profile page, which would have their clubs and lists

Reply:
Ok, we can discuss it more later.